cpp/src/barretenberg/honk/flavor/standard.hpp

```diff
@@ -0,0 +1,275 @@
+#pragma once
+#include <array>
+#include <concepts>
+#include <span>
+#include <string>
+#include <type_traits>
+#include <vector>
+#include "barretenberg/honk/pcs/commitment_key.hpp"
+#include "barretenberg/honk/sumcheck/polynomials/univariate.hpp"
+#include "barretenberg/ecc/curves/bn254/g1.hpp"
+#include "barretenberg/honk/transcript/transcript.hpp"
+#include "barretenberg/plonk/proof_system/proving_key/proving_key.hpp"
+#include "barretenberg/polynomials/evaluation_domain.hpp"
+#include "barretenberg/polynomials/polynomial.hpp"
+#include "barretenberg/proof_system/circuit_constructors/standard_circuit_constructor.hpp"
+#include "barretenberg/srs/reference_string/reference_string.hpp"
+#include "barretenberg/proof_system/flavor/flavor.hpp"
+
+namespace proof_system::honk::flavor {
+
+/**
+ * @brief Standard Honk
+ * @details We built this flavor first because it is the most basic. Because of this, it will remain useful for testing
+ * various constructions. Future variants may exist with varying: underlying curve (here we use BN254); commitment
+ * scheme (here we use Gemini + Shplonk + KZG); zero knowlege property (it's not implemented yet, but in the future we
+ * will be able to toggle it on or off).
+ *
+ */
+class Standard {
+  public:
+    using CircuitConstructor = StandardCircuitConstructor;
+    using FF = barretenberg::fr;
+    using Polynomial = barretenberg::Polynomial<FF>;
+    using PolynomialHandle = std::span<FF>;
+    using G1 = barretenberg::g1;
+    using GroupElement = G1::element;
+    using Commitment = G1::affine_element;
+    using CommitmentHandle = G1::affine_element;
```

Comment:
Eventually it could be a pointer. Here we just define the alias for uniformity (we DO want to use handles when we make the `ProverPolynomials` classes, in which case we use `std::span`s.).